test/integration/render-tests/icon-collision-group/default/style.json

```diff
@@ -0,0 +1,145 @@
+{
+    "version": 8,
+    "metadata": {
+        "test": {
+            "crossSourceCollisions": false,
+            "height": 128,
+            "width": 128,
+            "description": "Three collision groups of two layers each. Each group should show one label (overlapping with labels from other groups)"
+        }
+    },
+    "center": [
+        0,
+        0
+    ],
+    "zoom": 0,
+    "sources": {
+        "source1": {
+            "type": "geojson",
+            "data": {
+                "type": "FeatureCollection",
+                "features": [
+                    {
+                        "type": "Feature",
+                        "geometry": {
+                            "type": "Point",
+                            "coordinates": [
+                                0,
+                                0
+                            ]
+                        }
+                    }
+                ]
+            }
+        },
+        "source2": {
+            "type": "geojson",
+            "data": {
+                "type": "FeatureCollection",
+                "features": [
+                    {
+                        "type": "Feature",
+                        "geometry": {
+                            "type": "Point",
+                            "coordinates": [
+                                0,
+                                0
+                            ]
+                        }
+                    }
+                ]
+            }
+        },
+        "source3": {
+            "type": "geojson",
+            "data": {
+                "type": "FeatureCollection",
+                "features": [
+                    {
+                        "type": "Feature",
+                        "geometry": {
+                            "type": "Point",
+                            "coordinates": [
+                                0,
+                                0
+                            ]
+                        }
+                    }
+                ]
+            }
+        }
+    },
+  "sprite": "local://sprites/sprite",
+    "layers": [
+        {
+            "id": "defaultGroup1",
```

Comment:
Would this really use the "default" group?  My understanding is that with crossSourceCollisions: false, we're giving each source a named group.

Reply:
Updated the names of the tests and the IDs of these layers to get rid of the vestigial grouping logic.